src/components/Field/Field.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Field = ({
+  name, err, handleChange, validation, value, valid,
```

Comment:
```suggestion
  name, error, handler, validation, value, valid,
```
чем отличается valiation от valid?
если булиан, то isValid